kafka-1.0.x/src/main/scala/monix/kafka/KafkaConsumerObservable.scala

```diff
@@ -39,6 +40,8 @@ trait KafkaConsumerObservable[K, V, Out] extends Observable[Out] {
   protected def config: KafkaConsumerConfig
   protected def consumer: Task[KafkaConsumer[K, V]]
 
+  protected val isAcked = Atomic(true)
```

Comment:
I wonder if it could be just a `volatile`?

Reply:
Yeah, probably `volatile` should suffice here, `isAcked` only assigned _or_ checked